package.json

```diff
@@ -6,20 +6,13 @@
   ],
   "devDependencies": {
     "@changesets/cli": "^2.22.0",
-    "@liquality/bitcoin-esplora-api-provider": "^1.4.4",
-    "@liquality/bitcoin-js-wallet-provider": "^1.4.4",
-    "@liquality/bitcoin-networks": "^1.4.4",
     "@liquality/bitcoin-node-wallet-provider": "^1.4.4",
     "@liquality/bitcoin-rpc-provider": "^1.4.4",
```

Comment:
Can these two packages be removed as well? Along with everywhere they are used?

Reply:
Good question – I haven't migrated `@liquality/bitcoin-node-wallet-provider` over as it's been only used in CI/CD in our projects thus far.

Reply:
Just for full migration sake, will do so as well